### ISSUE TITLE
"Accessibility Tab Mode" background and text pure white on hc-dark theme

### ISSUE THREAD

**Issue by bgashler1** [Microsoft/vscode#3185]
We need to address the dialogue for how to use the "Accessibility Tab Mode."  It's pure white currently on hc-dark theme.  Should be a simple fix.

**Comment by bgashler1** [comment]
Duplicate #3586
